android/build.gradle

```diff
@@ -110,18 +110,19 @@ boolean CLIENT_SIDE_BUILD = resolveClientSideBuild()
 if (CLIENT_SIDE_BUILD) {
     configurations.maybeCreate("default")
 
-    def libInstanceCounter = 0;
-    String parsedLocation = "";
-    fileTree(rootDir.parent).visit { FileVisitDetails details ->
-        def path = details.file.path
-        if (path.contains("/react-native-reanimated/package.json")) {
-            libInstanceCounter++;
-            parsedLocation += "- " + path.replace("/package.json", "\n")
-        }
-        if (libInstanceCounter > 1) {
-            String exceptionMessage = "\nMultiple versions of Reanimated were detected. Only one instance of react-native-reanimated can be installed in a project. You need to resolve the conflict manually. Check out the documentation: https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/troubleshooting#multiple-versions-of-reanimated-were-detected \n\nConflict between: \n" + parsedLocation + "\n";
-            throw new Exception(exceptionMessage);
-        }
+    String parsedLocation = ""
+    def libInstancesCount = Files.walk(Paths.get(rootDir.parent))
```

Comment:
How long does it take? Visiting each file in the project can be slow in huge repos. Can we use something like glob instead?